### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54300]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
Pharmacy should live under “Programs” based on breadcrumb, but is not located there. The URL is also not in sync with breadcrumb.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/52672_Sara_040101.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

### Platform Recommendation
If Pharmacy is intended to live under programs, the page should be nested within that section of the left-nav. If not, the breadcrumb segment "Programs" should be removed for this page.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by sterkenburgsara** [comment]
@Dottisea as an update, I synced with @mnorthuis and we had a miscommunication. Since this breadcrumb problem is a true error, it is launch-blocking and will need to be resolved prior to launch. Details are above, but contact me if you have any questions. Thank you! // cc @shiragoodman

**Comment by davidmpickett** [comment]
This is a content configuration issue. A CMS Content Admin needs to move the Pharmacy pages to the root levels of the menu for their systems (rather than being nested under Programs). This matches the IA for all existing VAMC System Pharmacy Pages ([example](https://www.va.gov/chicago-health-care/pharmacy/)).

![Screen Shot 2023-02-28 at 12 51 41 PM](https://user-images.githubusercontent.com/51967950/221951375-dc2beba3-8334-41a6-9468-338f983758ce.png)

However, due to the uniqueness of how Lovell's menu is set up, it is unclear which of the 3 "Lovell Federal Health Care" options is correct. This is a good example of the type of information that should be included in the [KB article for CMS Content Admins](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11689) abut unique aspects of the Lovell parts of the VAMC product.

![Screen Shot 2023-02-28 at 12 52 16 PM](https://user-images.githubusercontent.com/51967950/221951524-e7929487-5b67-48a4-9938-a19e9b9dd9a5.png)
